mp/src/game/client/momentum/ui/HUD/hud_keypress.cpp

```diff
@@ -39,26 +39,22 @@ class CHudKeyPressDisplay : public CHudElement, public Panel
 
     CHudKeyPressDisplay(const char *pElementName);
 
-    bool ShouldDraw() OVERRIDE;
-    void OnThink() OVERRIDE;
-    void Paint() OVERRIDE;
-    void Init() OVERRIDE;
-    void Reset() OVERRIDE;
+    void ApplySchemeSettings(IScheme *pScheme) override;
+    bool ShouldDraw() override;
+    void OnThink() override;
+    void Paint() override;
+    void Init() override;
+    void Reset() override;
     void DrawKeyTemplates();
 
-    void ApplySchemeSettings(IScheme *pScheme) OVERRIDE
-    {
-        Panel::ApplySchemeSettings(pScheme);
-        SetBgColor(Color(0, 0, 0, 1)); // empty background, 1 alpha (out of 255) so game text doesn't obscure our text
-    }
-
   protected:
     CPanelAnimationVar(HFont, m_hTextFont, "TextFont", "Default");
     CPanelAnimationVar(HFont, m_hWordTextFont, "WordTextFont", "Default");
     CPanelAnimationVar(HFont, m_hCounterTextFont, "CounterTextFont", "Default");
     CPanelAnimationVar(Color, m_Normal, "KeyPressedColor", "Panel.Fg");
     CPanelAnimationVar(Color, m_darkGray, "KeyOutlineColor", "Dark Gray");
     CPanelAnimationVar(Color, m_Disabled, "KeyDisabledColor", "Red");
+    CPanelAnimationVar(Color, m_BgColor, "BgColor", "0 0 0 1"); // empty background, 1 alpha so game text doesn't obscure our text
```

Comment:
I'm not even sure what the original intent was with the 1 alpha, but if the background color needs set there's `BgColor_override` inside every panel

Reply:
Not sure why either, works just fine with 0 alpha.